src/main/kotlin/no/nav/syfo/api/model/VurderingResponseDTO.kt

```diff
@@ -30,6 +32,7 @@ data class VurderingResponseDTO private constructor(
             document = vurdering.document,
             varsel = vurdering.varsel?.let { VarselDTO.createFromVarsel(it) },
             gjelderFom = vurdering.gjelderFom,
+            vurderingInitiertAv = if (vurdering is Vurdering.AvslagUtenForhandsvarsel) vurdering.vurderingInitiertAv else null,
```

Comment:
Hmm, jeg tenkte kanskje at dette er logikk som burde ligge i domenet? At man her bare skulle kunne returnert `vurdering.vurderingInitiertAv` og så vil den være `null` eller ikke i de riktige tilfellene 🤔

Reply:
Aha, mulig jeg bare ikke forstår sealed classes godt nok egentlig 👍🏼
Så alternativene er enten å gjøre det i DTOen eller å ha en lignende funksjon som man har for `arsak` nå?

Reply:
Av det jeg kommer på, ja 😊 Ser heller ikke egentlig på dette som logikk. Hadde propertien eksistert flatt på `sealed class`en så ville dette vært det ekvivalente 👇
```suggestion
            vurderingInitiertAv = vurdering?.vurderingInitiertAv,
```